src/domains/shared-app/api/session-wallet/v1/_base_policy.xml.tpl

```diff
@@ -1,225 +1,166 @@
-          </when>
-      </choose>
-      <set-variable name="pmSession" value="@(((IResponse)context.Variables["pm-session-body"]).Body.As<JObject>())" />
-      <!-- Session PM END-->
-      <!-- user fiscal code tokenization with PDV START -->
-      <set-variable name="userFiscalCode" value="@{
-          String userFiscalCode = ((JObject)context.Variables["pmSession"])?["data"]?["user"]?["fiscalCode"]?.ToString();
-          return userFiscalCode;
-      }" 
-      />
-      <choose>
-          <when condition="@(String.IsNullOrEmpty((String)context.Variables["userFiscalCode"]))">
-              <return-response>
+        <when condition="@(((IResponse)context.Variables["user-auth-body"]).StatusCode == 401)">
+            <return-response>
+                <set-status code="401" reason="Unauthorized" />
+                <set-header name="Content-Type" exists-action="override">
+                    <value>application/json</value>
+                </set-header>
+                <set-body>
+                    {
+                        "title": "Unauthorized",
+                        "status": 401,
+                        "detail": "Invalid session token"
+                    }
+                </set-body>
+            </return-response>
+        </when>
+        <when condition="@(((IResponse)context.Variables["user-auth-body"]).StatusCode != 200)">
+            <return-response>
                 <set-status code="502" reason="Bad Gateway" />
                 <set-header name="Content-Type" exists-action="override">
-                  <value>application/json</value>
+                    <value>application/json</value>
                 </set-header>
                 <set-body>
                     {
                         "title": "Error starting session",
                         "status": 502,
-                        "detail": "Cannot tokenize user fiscal code: PM start-session fiscalCode is null"
+                        "detail": "There was an error while getting user info"
                     }
                 </set-body>
-              </return-response>
-          </when>
-          <otherwise>
-              <!-- Post Token PDV : START-->
-              <send-request ignore-error="true" timeout="10" response-variable-name="pdv-token" mode="new">
-                <set-url>${pdv_api_base_path}/tokens</set-url>
-                <set-method>PUT</set-method>
-                <set-header name="x-api-key" exists-action="override">
-                    <value>{{wallet-session-personal-data-vault-api-key}}</value>
+            </return-response>
+        </when>
+      </choose>
+      <set-variable name="userAuthBody" value="@(((IResponse)context.Variables["user-auth-body"]).Body.As<JObject>())" />
+      <set-variable name="userFiscalCode" value="@{
+        String userFiscalCode = ((JObject)context.Variables["userAuthBody"])?["fiscal_code"]?.ToString();
+        return userFiscalCode;
+    }" 
+    />
```

Comment:
```suggestion
      <set-variable name="userFiscalCode" value="@(@((string)(((JObject)context.Variables["userAuthBody"]).Body.As<JObject>())["fiscal_code"]))"/>
```

Reply:
Fixed in 7db3c623066d98e9b3733dadfc9ecf923f0d8f39